.drone.yml

```diff
@@ -42,7 +42,7 @@ pipeline:
     image: nextcloudci/php5.6:php5.6-3
     environment:
       - APP_NAME=impersonate
-      - CORE_BRANCH=master
+      - CORE_BRANCH=stable12
```

Comment:
for 13 i thought?

Reply:
yes, but this branch supports both 12 and 13. so a 12 test on an ancient php version makes sense, no?

Reply:
the other branches must go against stable13 though